mypy/util.py

```diff
@@ -12,6 +12,17 @@
 default_python2_interpreter = ['python2', 'python', '/usr/bin/python']
 
 
+def split_module_names(mod_name: str) -> Iterable[str]:
+    """Yields the module and all parent module names.
+
+    So, if `mod_name` is 'a.b.c', this function will yield
+    ['a.b.c', 'a.b', and 'a']."""
```

Comment:
General tip: when a docstring is > 1 line, the final """ should go on a line by itself. (PEP 257)